echo-web/config/echo.yml

```diff
@@ -2,16 +2,24 @@ server:
   port: 8089
 
 spinnaker:
+  inMemory:
+    enabled: true
   cassandra:
     cluster: 'spinnaker'
     keyspace: 'echo'
-  baseUrl: 'http://localhost:9000'
+  baseUrl: ${services.deck.baseUrl:http://localhost:9000}
 
 front50:
-  baseUrl: 'http://localhost:8080'
+  enabled: ${services.front50.enabled:false}
```

Comment:
It looks like we never actually read this in, so we should probably omit it to avoid confusion...from what I can tell `front50` is *always* enabled.